charts/newrelic-infrastructure/templates/kubelet/daemonset-windows.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if and .Values.kubelet.enabled .Values.kubelet.enableWindows }}
+{{- range $key, $windowsList := .Values.kubelet.windowsOsList }}
```

Comment:
it seems like we can just range over the values {{- range .Values.kubelet.windowsOsList }}

Reply:
Done